src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -24,6 +24,7 @@
 import urllib.parse
 
 from google.auth import exceptions
+from googleapiclient.http import HttpRequest
```

Comment:
Can you import `googleapiclient.http` and then use `googleapiclient.http.HttpRequest`? It better matches the styleguide.